### PR TITLE
Show my group

### PR DESCRIPTION
**Pull request by PikkuHukka**
Shows the group in registration details page. If there is no instructor, backend will return "Tällä ryhmällä ei ole vielä ohjaajaa" string in {group.instructor} Otherwise it will be first_names + " " + last_name string.

### CODE REVIEW

File: src/components/PeerReviewPage.js [hunk 1]
```diff
@@ -30,29 +30,27 @@ class PeerReview extends React.Component {
     try {
       const group = await groupManagementService.getByStudent()
       if (group) {
-        if (group.id !== -100) {
```
Comment (cxcorp) [inline src/components/PeerReviewPage.js]: Why is the -100 not needed anymore? Does the backend still serve -100? Is it needed anywhere? If not, can it be removed from the backend?

File: src/components/RegistrationDetailsPage.js [hunk 2]
```diff
@@ -154,8 +155,42 @@ const RegistrationAnswers = ({ questions }) => {
...
+    )
+  }
+}
+
 class RegistrationDetailsPage extends React.Component {
+  state = { groupDetails: null }
+
+  async componentDidMount() {
+    const myGroup = await groupManagementService.getByStudent()
+    this.setState({
```
Comment (cxcorp) [inline src/components/RegistrationDetailsPage.js]: Why is this in local state and not in redux state? Redux state is still passed in via mapStateToProps but it's never used?

File: src/components/RegistrationDetailsPage.js [hunk 3]
```diff
@@ -154,8 +155,42 @@ const RegistrationAnswers = ({ questions }) => {
   )
 }
 
+const GroupDetails = ({ groupDetails }) => {
+  if (!groupDetails) {
+    return (
+      <div>
+        <h2>This user is currently not part of any group.</h2>
```
Comment (cxcorp) [inline src/components/RegistrationDetailsPage.js]: Would a more user-friendly copy be "You are currently not part of any group"?

File: src/components/RegistrationDetailsPage.js [hunk 4]
```diff
@@ -154,8 +155,42 @@ const RegistrationAnswers = ({ questions }) => {
...
+      </div>
+    )
+  } else {
+    return (
+      <div>
+        <h2>{groupDetails.groupName}</h2>
+        {groupDetails.students.map((member, index) => {
+          return (
+            <p>
```
Comment (cxcorp) [inline src/components/RegistrationDetailsPage.js]: Remember `key={member.student_number}` or something

File: src/components/RegistrationDetailsPage.js (hunk 3 above)
Comment (akiutoslahti) [reply src/components/RegistrationDetailsPage.js]: How about not showing group details at all if user is not in group?

File: src/components/RegistrationDetailsPage.js (hunk 4 above)
Comment (akiutoslahti) [reply src/components/RegistrationDetailsPage.js]: or index

File: src/components/RegistrationDetailsPage.js (hunk 2 above)
Comment (akiutoslahti) [reply src/components/RegistrationDetailsPage.js]: instructor attribute could be fetched on login and stored to redux state, so it does not need to be refetched from backend on every page.

File: src/components/PeerReviewPage.js (hunk 1 above)
Comment (PikkuHukka) [reply src/components/PeerReviewPage.js]: Backend no longer serve -100. Therefore this check can safely be removed. Backend will now serve null group which is checked by the line above.

**Pull request merged**